yt/admin/dashboard_generator/yt_dashboard_generator/helpers.py

```diff
@@ -34,3 +34,98 @@ def pretty_dump_fixed_indent(x, offset=""):
 
 def pretty_print_fixed_indent(x):
     print(pretty_dump_fixed_indent(x))
+
```

Comment:
Two emply lines between functions (and you can run `ya make -t` to check linter warnings)